### PR TITLE
add performance test baselines for iPhone 5s & iPad Mini 2

### PR DESCRIPTION
**Pull request by jpsim**
These are now connected to CI so they need performance baselines. /cc @tgoyne

### CODE REVIEW

**Comment by jpsim** [comment]
@tgoyne

File: build.sh [hunk 1]
```diff
@@ -218,6 +220,16 @@ test_ios_devices() {
     configuration="$3"
     failed=0
     for device in "${serial_numbers[@]}"; do
+        if [[ "$device" = "07510d12c09c53e44b3682dd0367934f82cfab9c" ]]; then
```
Comment (tgoyne) [inline build.sh]: Doesn't this result in the swift devices tests job just not running if the VM with the ios7 device gets picked? Removing it as on option using a tag for ios 8+ in jenkins seems better.

Comment (jpsim) [inline build.sh]: You're right, I made this PR before we decided to parallelize the objc and swift device tests. I'll remove this change.

Comment (jpsim) [inline build.sh]: Done.

**Pull request merged**